link_checker/check.sh

```diff
@@ -141,8 +151,8 @@ test_link () {
 			return 1 # Interrupted
 
 		elif [[ ! "$curl_status" -eq 0 ]]; then
-			[[ $MARKDOWN ]] && echo "* [ ] --- $text $link (cURL exited with: $curl_status)"
-			[[ ! $QUIET ]] && echo >&2 "🤷 [---] Checking $hostport failed with error code $curl_status"
+			[[ $MARKDOWN ]] && echo "* [ ] --- $text $link (CURLcode: $curl_status)"
+			[[ ! $QUIET ]] && echo >&2 "🤷 [---] Checking $hostport failed with CURLcode $curl_status"
```

Comment:
Should `$hostport` be `${hostport}`, at the very least for consistency with the above branches?

Reply:
Sure, +1 for the consistency! 

Though, it's not really required here, nor in any of the other cases; afaik, the `${}` syntax is for string interpolation safety.  Eg., it's to tell the difference between something like `echo aaa${hostport}aaa` and `echo aaa$hostportaaa`. You can make the same comment about `$curl_status`, `$text`, `$link`, etc... but since none of the surrounding characters are alphanumeric right now it's not a problem. :)

Reply:
Oh, and it's of course required for interacting with array and associative array elements, a-là `${some_array[3]}`!